core/src/main/java/com/opensymphony/xwork2/config/impl/AbstractMatcher.java

```diff
@@ -36,6 +36,9 @@
  * @since 2.1
  */
 public abstract class AbstractMatcher<E> implements Serializable {
+
+    private static final Logger LOG = LogManager.getLogger(AbstractMatcher.class);
```

Comment:
This looks like it introduces a second private `Logger` reference ?  There is already:
```
 private static final Logger log = LogManager.getLogger(AbstractMatcher.class); 
```
Maybe the intention was to rename `log` to `LOG` and update all the logger calls ?

Reply:
Right, fixed :)